### PR TITLE
Scrollable history fixes

### PR DESCRIPTION
**Pull request by IhateTrains**
- fixed obvious charID typos
- scrollable history fixes (mostly Italy area)
- code cleaning

### CODE REVIEW

File: WTWSMS/history/titles/c_carcassonne.txt [hunk 1]
```diff
@@ -29,21 +29,32 @@
 507.1.1={ 
 	holder=190067 #Gaisalaiks
 }
-508.1.1={ 
-	holder=63 #Theodoric 
-}
-510.1.1={
-	holder=154003 # Petrus Marcellinus Felix Liberius
-	liege = "k_italy"
-}
```
Comment (loup99) [inline WTWSMS/history/titles/c_carcassonne.txt]: Why did you remove these?

File: WTWSMS/history/titles/c_narbonne.txt [hunk 2]
```diff
@@ -29,44 +29,32 @@
...
-	holder = 30015 # Sigebert Meroving 
-}
-614.1.1={
-	holder = 30029 # Chlothar II Meroving 
+572.1.1={ 
+	holder=212873 #Liubagilds
 }
+586.1.1={ 
+	holder=36556 #Reccared
+}
```
Comment (loup99) [inline WTWSMS/history/titles/c_narbonne.txt]: Same as previous.

File: WTWSMS/history/titles/c_rosello.txt [hunk 3]
```diff
@@ -36,7 +36,25 @@
...
 570.1.1={ 
 	holder=36662 #Gundemar
-	liege = k_visigoths 
+	liege = k_visigoths
+}
+572.1.1={ 
+	holder=212873 #Liubagilds
+}
+586.1.1={ 
+	holder=36556 #Reccared
```
Comment (loup99) [inline WTWSMS/history/titles/c_rosello.txt]: See previous.

File: WTWSMS/history/titles/k_saxony.txt [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+	law = succ_gavelkind
 	law = cognatic_succession
 }
 450.1.1={
-	holder = 45
+	holder = 45 # Hadugato, fictional
 }
 497.1.1={
-	holder = 30037 
+	holder = 30037 # Gunther, fictional
```
Comment (loup99) [inline WTWSMS/history/titles/k_saxony.txt]: I might be wrong on these, but I think it might be based upon splitting up historical rulers.

Comment (IhateTrains) [reply WTWSMS/history/titles/k_saxony.txt]: Done.

File: WTWSMS/history/titles/c_carcassonne.txt (hunk 1 above)
Comment (IhateTrains) [reply WTWSMS/history/titles/c_carcassonne.txt]: My bad. Not all sources mention this, but Ostogoths did indeed capture the city. The date (508) seems a bit too early, however 😮

Comment (IhateTrains) [reply WTWSMS/history/titles/c_carcassonne.txt]: Wasn't it about 518?

Comment (IhateTrains) [reply WTWSMS/history/titles/c_carcassonne.txt]: OK, should be fixed now.

**Comment by IhateTrains** [comment]
Are Carcassone/Rosello/Narbonne OK now? Are there any more problems?

File: WTWSMS/history/titles/c_carcassonne.txt [hunk 5]
```diff
@@ -30,20 +30,39 @@
...
+554.1.1={ 
+	holder=36555 #Athanagild 
+}
+567.1.1={ 
+	holder=212872 #Liuba
+}
 570.1.1={ 
 	holder=36662 #Gundemar
-	liege = k_visigoths 
+	liege = k_visigoths
```
Comment (loup99) [inline WTWSMS/history/titles/c_carcassonne.txt]: If k_visigoths are the liege from 511.1.1, they don't need to become it again in 570.1.1.

File: WTWSMS/history/titles/c_malta.txt [hunk 6]
```diff
@@ -9,6 +9,7 @@
 }
 491.1.1 = { #Cassiodorus
 	holder=154006
+	liege = "k_italy"
 }
```
Comment (loup99) [inline WTWSMS/history/titles/c_malta.txt]: Wait, is the 491.1.1 block new? In 428.1.1 Malta is lead by the Vandals.

Comment (IhateTrains) [reply WTWSMS/history/titles/c_malta.txt]: Cassiodorus was added here by @tannerflick4

File: WTWSMS/history/titles/c_carcassonne.txt (hunk 5 above)
Comment (IhateTrains) [reply WTWSMS/history/titles/c_carcassonne.txt]: It's an unharmful leftover, but I will remove it nonetheless

Comment (IhateTrains) [reply WTWSMS/history/titles/c_carcassonne.txt]: Done

File: WTWSMS/history/titles/c_malta.txt (hunk 6 above)
Comment (IhateTrains) [reply WTWSMS/history/titles/c_malta.txt]: I don't know if Cassiodorus should be here. From what I know Ostrogoths seized Lilybaeum (c_trapani) and probably Malta in 535 during Roman expedition against the Vandals.

Comment (tannerflick4) [reply WTWSMS/history/titles/c_malta.txt]: I am not sure who owned Malta in 491. I probably gave him Malta since it was in the realm he was ruler of as a vassal of Italy. So if @IhateTrains knows more, then its ok to fix this county.

Comment (IhateTrains) [reply WTWSMS/history/titles/c_malta.txt]: By "the realm he was ruler of" you mean Sicily?

**Comment by tannerflick4** [comment]
Yes.

File: WTWSMS/history/titles/c_malta.txt (hunk 6 above)
Comment (IhateTrains) [reply WTWSMS/history/titles/c_malta.txt]: OK, I will fix c_malta and c_trapani then.

**Comment by tannerflick4** [comment]
Thank you! I forgot about the Ostrogoths taking those over that year.

File: WTWSMS/history/titles/c_malta.txt (hunk 6 above)
Comment (IhateTrains) [reply WTWSMS/history/titles/c_malta.txt]: Small correction on my post on history: Malta was taken by Romans, not Ostrogoths.

**Comment by tannerflick4** [comment]
Oh okay. No prob

**Comment by IhateTrains** [comment]
The historical issues should be addressed now :)

File: WTWSMS/history/titles/c_malta.txt (hunk 6 above)
Comment (loup99) [reply WTWSMS/history/titles/c_malta.txt]: Then I say revert both, neither of them should have been changed.

Comment (IhateTrains) [reply WTWSMS/history/titles/c_malta.txt]: Hasn't it been addressed with bf6fe53?

Comment (loup99) [reply WTWSMS/history/titles/c_malta.txt]: Yes.

**Comment by IhateTrains** [comment]
gonna make a fresh PR with the same changes
